commands/cms/lighthouseScore.ts

```diff
@@ -28,7 +28,7 @@ const DEFAULT_TABLE_HEADER = [
 ];
 
 exports.command = 'lighthouse-score [--theme]';
-exports.describe = false; // i18n(`${i18nKey}.describe`);
+exports.describe = false;
```

Comment:
I think we can unhide this command now. Maybe check with @brandenrodgers

Reply:
The last time this came up, the answer was to let the CMS team determine the future of the commands.  I'm not sure where they landed on it